### PR TITLE
Add option to ignore masking for given Namespaces

### PR DESCRIPTION
**Pull request by Kevin-Bronsdijk**
I really like this idea of masking logs to avoid sensitive data being exposed within the logs. However, we run into several performance issues and therefore would like to introduce a way to exclude namespaces from being masked because we know they won't expose properties to be masked. This greatly improved the performance. 

This is just an idea, let me know what you think!

### CODE REVIEW

File: src/Masking.Serilog/Masking.Serilog.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <Title>Masking.Serilog</Title>
     <Description>Masking sensitive information during logging to Serilog by hiding individual properties.</Description>
     <PackageProjectUrl>https://github.com/evjenio/masking.serilog</PackageProjectUrl>
-    <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```
Comment (Kevin-Bronsdijk) [inline src/Masking.Serilog/Masking.Serilog.csproj:11]: based on https://spdx.org/licenses/

File: src/Masking.Serilog/ByMasking/MaskingOptions.cs [hunk 2]
```diff
@@ -34,5 +34,10 @@ public class MaskingOptions
         /// A flag to specify whether static properties should be included when Destructuring the object.
         /// </summary>
         public bool ExcludeStaticProperties { get; set; } = false;
+        
+        /// <summary>
+        /// List of namespaces for which masking will be ignored.
+        /// </summary>
+        public List<string> Namespaces { get; } = new List<string>();
```
Comment (evjenio) [inline src/Masking.Serilog/ByMasking/MaskingOptions.cs]: What if we call it `IgnoredNamespaces` to make it more clear?

File: src/Masking.Serilog/Masking.Serilog.csproj (hunk 1 above)
Comment (evjenio) [reply src/Masking.Serilog/Masking.Serilog.csproj:11]: Ok thanks

**Comment by evjenio** [comment]
Hello @Kevin-Bronsdijk 
This is looking good. Could you check my comments? Thank you!

File: src/Masking.Serilog/ByMasking/MaskingOptions.cs (hunk 2 above)
Comment (Kevin-Bronsdijk) [reply src/Masking.Serilog/ByMasking/MaskingOptions.cs]: Makes perfect sense. Will change this as suggested,

**Review by evjenio — APPROVED** [review]

**Pull request merged**